src/APICalls.js

```diff
@@ -0,0 +1,12 @@
+export default function getData(dataset) {
```

Comment:
This was a doozy for a second there, so glad we got it working and in its own file, yay us!